lib/iex/lib/iex/evaluator.ex

```diff
@@ -220,6 +243,7 @@ defmodule IEx.Evaluator do
       env: env,
       server: server,
       history: history,
+      last_op: :other,
```

Comment:
Should no longer be here. :)

Reply:
It’s evaluator’s state. Not `iex` state. And I need it to use a short notation `%{state | last_op: last_op}` down the rabbit hole.